integrations/appetize_integration_demo_project/codemagic.yaml

```diff
@@ -0,0 +1,96 @@
+workflows:
+  appetize-android-workflow:
+    name: Appetize Android Workflow
+    instance_type: linux_x2
+    cache:
+      cache_paths:
+        - $HOME/.gradle/caches
+        - $CM_BUILD_DIR/build
+    environment:
+      groups:
+        - appetize # <-- Includes (API_TOKEN)
+      vars:
+        PACKAGE_NAME: "com.mohammed.codemagic_demo" # <-- Put your package name here
+        APP_PUBLIC_KEY: "**************************" # <-- Put your App public key here
+      android_signing:
+        - codemagic_demo
+      flutter: stable
+    scripts:
+      - name: Set up key properties
+        script: |
+          # echo $CM_KEYSTORE | base64 --decode > $CM_KEYSTORE_PATH # Not required if using team code signing identities
+          # The following script creates key.properties file and stores the credentials in it. As we configure code signing in app/build.gradle file, the following part is unnecessary unless code signing is configured differently as explained in the documentation: https://docs.codemagic.io/flutter-code-signing/android-code-signing/
+          cat >> "$CM_BUILD_DIR/android/key.properties" <<EOF
+          storePassword=$CM_KEYSTORE_PASSWORD
+          keyPassword=$CM_KEY_PASSWORD
+          keyAlias=$CM_KEY_ALIAS
+          storeFile=$CM_KEYSTORE_PATH
+          EOF
+      - name: Set up local.properties
+        script: |
+          echo "flutter.sdk=$HOME/programs/flutter" > "$CM_BUILD_DIR/android/local.properties"
+      - name: Get Flutter packages
+        script: |
+          flutter packages pub get
+      - name: Build apk with Flutter
+        script: |
+          flutter build apk --release
+    artifacts:
+      - build/**/outputs/apk/**/*.apk
+    publishing:
+      scripts:
+        - name: Upload to appetize
+          script: |
+            artifactPath="$CM_BUILD_OUTPUT_DIR/app-release.apk"
+            echo $(curl --location --request POST 'https://'$API_TOKEN'@api.appetize.io/v1/apps/'$APP_PUBLIC_KEY'' --form 'file=@"'$artifactPath'"')
+  appetize-ios-workflow:
+    name: iOS Workflow
+    instance_type: mac_pro
+    max_build_duration: 120
+    triggering:
+      events:
+        - push
+      branch_patterns:
+        - pattern: "*"
+          include: true
+    environment:
+      groups:
+        - appetize # Includes (API_TOKEN)
+        - app_store_credentials # <-- (Includes APP_STORE_CONNECT_ISSUER_ID, APP_STORE_CONNECT_KEY_IDENTIFIER, APP_STORE_CONNECT_PRIVATE_KEY)
+        - certificate_credentials # <-- (Includes CERTIFICATE_PRIVATE_KEY - Put your Certificate Private Key here)
+      vars:
+        XCODE_WORKSPACE: "Runner.xcworkspace"
+        XCODE_SCHEME: "Runner"
+        BUNDLE_ID: "com.mohammed.codemagicDemo" # <-- Put your bundle id here
```

Comment:
try to use bundle ids that reflect the nature of the sample e.g. io.codemagic.appetize